bin/node-template/node/Cargo.toml

```diff
@@ -37,5 +38,4 @@ sc-basic-authorship = { path = "../../../client/basic-authorship", version = "0.
 node-template-runtime = { version = "2.0.0-alpha.5", path = "../runtime" }
 
 [build-dependencies]
-vergen = "3.0.4"
-build-script-utils = { version = "2.0.0-alpha.5", package = "substrate-build-script-utils", path = "../../../utils/build-script-utils" }
+sc-cli = { version = "0.8.0-alpha.4", path = "../../../client/cli" }
```

Comment:
You mean I should move it to substrate-build-script-utils? (That's fine by me)

Reply:
Note: sc-cli OR structopt still needs to be in the build dependencies because that's how the shell completions are generated (in bin/node/cli).

Using structopt directly is fine (even if the version don't match) but if sc-cli gets a structopt upgrade to a major release, the structopt build dependency will need to get also the upgrade.

Reply:
Note: actually we need:
- sc-cli for the ShareParams, ImportParams, etc...
- structopt because the derive cannot be imported from sc-cli https://github.com/TeXitoi/structopt/issues/339
- and now substrate-build-script-utils (that's the only one we could remove if we just re-export what we need)